docs/quantum-mechanic/command-reference.md

```diff
@@ -1,76 +1,352 @@
+
+---
+#### `sub help`
+Displays a help menu as a Slack browsable form of the Quick Reference section of this page.
+
+---
+#### `sub jenkins build`
+This will kick off a Jenkins dev pipeline build for a selected application. That is it will run the build associated to the standard Jenkinsfile for an Application, and not the additional deployment jobs associated to the Application.
+
+---
+#### `sub link application`
+This will prompt a user for details about an Application they wish to allow Subatomic to manage. This will require an existing initialised repository in the Bitbucket project associated to the Application's intended owning Subatomic Project. This creates the metadata representation of the Application within the Subatomic database. Once the Application has been created the user needs to decide how Subatomic should treat the Application. See [sub configure package](#sub-configure-package) for more details.
+
+---
+#### `sub link bitbucket project`
+When creating a Project in Subatomic, the Subatomic Project needs to be associated to a Bitbucket project. This command is used to do this linking. In order to link the Bitbucket project to the Subatomic Project the user needs to provide the Bitbucket project key. Instructions on finding this Bitbucket project key can be found in the [FAQ](../FAQ.md#sub-link-bitbucket-project).
+
+---
+#### `sub link library`
+This will prompt a user for details about a Library they wish to allow Subatomic to manage. This will require an existing initialised repository in the Bitbucket project associated to the Application's intended owning Subatomic Project. This creates the metadata representation of the Library within the Subatomic database. Once the Library has been created the user needs to decide how Subatomic should treat the Library. See [sub configure package](#sub-configure-package) for more details.
+
+---
+#### `sub link team channel`
+This command allows a user to link their Team to a new Slack channel for notification purposes. The Slack channel needs to exist for this command to work correctly. This is particularly useful if the currently associated Team channel was renamed, if you want to use a private slack channel(\*) for the team notifications, or if you have an existing channel you wish to use for the notifications and Subatomic control flow interaction. When linking a Team to a new Slack channel the following actions are performed:
+1. Invite the Members of the Team to the Slack channel.
+2. Invite the Atomist Bot to the Team Channel.
+
+The Atomist Bot should present a generic message about creating a DevOps environment whenever it joins a new channel that is associated to a Team.
+
+**\*Note:** If you want to use a private Slack channel, the Atomist bot cannot be invited automatically. You will need to manually invite the bot the to channel using `\invite @Atomist` once you have linked the Team to the Slack channel.
```

Comment:
Why does this Note have \*

Reply:
It is an exception to the normal flow as referenced by "if you want to use a private slack channel(\*)"